tpl/bootstrap.php

```diff
@@ -66,7 +66,25 @@
 		<meta name="theme-color" content="#ffe57e" />
 	</head>
 	<body role="document">
-		<nav class="navbar navbar-default navbar-static-top">
+        <!-- Modal decryption password request -->
+        <div class="modal fade" id="passwordModal" role="dialog">
+          <div class="modal-dialog">
+            <!-- Modal content-->
+            <div class="modal-content">
+              <div class="modal-body">
+            <form role="form" id="passwordForm">
+              <div class="form-group">
+                <label for="psw"><span class="glyphicon glyphicon-eye-open"></span> <?php echo I18n::_('Please enter the password for this paste:')?></label>
+                <input type="password" class="form-control" id="decryptPassword" placeholder="Enter password" autofocus>
+              </div>
+              <!-- <button type="submit" class="btn btn-success btn-block"><span class="glyphicon glyphicon-off"></span>Decrypt</button> -->
```

Comment:
Could you get this button to work?
There should really be a "Decrypt" (or similar) button in the UI.